app/src/main/res/values-land/dimens.xml

```diff
@@ -428,7 +428,14 @@
   <dimen name="topic_lessons_title_margin_start">100dp</dimen>
   <dimen name="topic_lessons_title_margin_end">100dp</dimen>
 
+  <!-- ProfileProgressRecentlyPlayedStoryCard -->
+  <dimen name="profile_progress_recently_played_story_card_layout_margin_start">16dp</dimen>
+  <dimen name="profile_progress_recently_played_story_card_layout_margin_end">16dp</dimen>
+  <dimen name="profile_progress_recently_played_story_card_layout_margin_top">24dp</dimen>
+  <dimen name="profile_progress_recently_played_story_card_layout_margin_bottom">24dp</dimen>
```

Comment:
Bottom margin should be `0dp` for landscape please check this and confirm once again.